### ISSUE TITLE
[translation] Update FR messages for CLI resources

### ISSUE THREAD

**Issue by aalmiray** [jreleaser/jreleaser#971]
Please check the translation files you want to add/fix/update:

 - [ ] CLI banner: (plugins/jreleaser/src/main/resources/org/jreleaser/cli/Banner*.properties)
 - [x] CLI bundle: (plugins/jreleaser/src/main/resources/org/jreleaser/cli/Messages*.properties)
 - [ ] core bundle: (core/jreleaser-utils/src/main/resources/org/jreleaser/bundle/Messages*.properties)

A list of language codes can be found at https://sda.berkeley.edu/man41h/localelist.htm

The following keys must be updated:

```
jreleaser.config.announce            = Display announce configuration.
jreleaser.config.changelog           = Display changelog configuration.
jreleaser.config.download            = Display download configuration.

jreleaser.release.author                          = The project authors. Repeatable.
jreleaser.release.project-copyright               = The project copyright.
jreleaser.release.project-description             = The project description.
jreleaser.release.project-inception-year          = The project inception year.
jreleaser.release.project-stereotype 

###############################################################################
# Deploy
###############################################################################
# header
jreleaser.deploy.usage.headerHeading =
jreleaser.deploy.usage.header        = Deploys all staged artifacts.

###############################################################################
# Download
###############################################################################
# header
jreleaser.download.usage.headerHeading =
jreleaser.download.usage.header        = Download assets.

###############################################################################
# Json-Schema
###############################################################################
# header
jreleaser.json-schema.usage.headerHeading =
jreleaser.json-schema.usage.header        = Generate JSON schema.
```

**Comment by adriens** [comment]
-  CLI banner: [Banner_fr.properties](https://github.com/jreleaser/jreleaser/blob/main/plugins/jreleaser/src/main/resources/org/jreleaser/cli/Banner_fr.properties)  seems to be already :ok:

**Comment by adriens** [comment]
:point_up: I cannot find : 

```
 (core/jreleaser-utils/src/main/resources/org/jreleaser/bundle/Messages*.properties)
```
Did I miss something ? Should I watch a specifi branch ?

![image](https://user-images.githubusercontent.com/5235127/193546141-42528105-c2f3-4da4-acb5-d819337d93e0.png)

**Comment by aalmiray** [comment]
Released in v1.4.0 -> https://github.com/jreleaser/jreleaser/releases/tag/v1.4.0
